gamedata/myz/scrap.en.list

```diff
@@ -247,7 +247,7 @@ Spade
 Spoon, wood
 Tablecloth
 Table knife
-Telephone diretory
+Telephone directory
```

Comment:
Wow, did you really check all entries in this list? :O

Reply:
I didn't check all the entries, the ones at the top i just replaced with the ones from the source book. But i did have to translate those last entries by hand, as they aren't from any particular book.